storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst

```diff
@@ -0,0 +1,56 @@
+.. _rn.8-0-8:
+
+================================================================================
+|pxb.name| |release|
+================================================================================
+
+|percona| is glad to announce the release of |pxb.name| |release| on |date|.
+Downloads are available from our `download site
+<https://www.percona.com/downloads/Percona-XtraBackup-LATEST/>`_ and from
+:ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+|pxb.name| enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot
+tolerate long periods of downtime. Offered free as an open source solution, it
+drives down backup costs while providing unique features for |MySQL| backups.
+
+All |percona| software is open-source and free.
+
+New Features and Improvements
+================================================================================
+
+- Support log archiving feature in PXB 8.0. More information in
+  :jirabug:`PXB-1912` and in the `Redo Log section of MySQL documentation
+  <https://dev.mysql.com/doc/refman/8.0/en/innodb-redo-log.html>`_
+- Two options (`–-backup-lock-timeout` and `-–backup-lock-retry-count`) were
+  added to enable the configuring of the timeout for acquiring metadata locks in
+  ``FLUSH TABLES WITH READ LOCK``, ``LOCK TABLE FOR BACKUP``, and ``LOCK BINLOG
+  FOR BACKUP`` statements. More information in :jirabug:`PXB-1914`
+
+Bugs Fixed
+================================================================================
+
+- An encrypted table could not be restored when ``ADD INDEX`` or ``DROP INDEX``
+  commands had been run on the table. Bug fixed :jirabug:`PXB-1905`
+- In some cases ``xtrabackup --prepare`` could fail to decrypt a table but
+  reported that the operation completed ok. Bug fixed :jirabug:`PXB-1936`
+- ``xtrabackup --move-back`` did not complete successfully when the encrypted
+  binlog file. Bug fixed :jirabug:`PXB-1937`.
+- |pxb.name| could crash during the prepare stage when making an incremental
+  backup when a multi valued index was being added added or dropped for JSON
+  data. Bug fixed :jirabug:`PXB-1913`.
+- A backup could fail when the database was on a fast storage and the redo
```

Comment:
no need RN. it was a bug in new feature which we release in this version

Reply:
Removed